packages/SidePanel/src/components/Dialog/Dialog.js

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { dialogStyles, dialogContentStyles } from "./Dialog.styles";
+
+const propTypes = {
+  children: PropTypes.node.isRequired,
+  footer: PropTypes.node,
+  header: PropTypes.node,
+  isInline: PropTypes.bool,
+  offsetY: PropTypes.number,
+  onAnimationEnd: PropTypes.func.isRequired,
+  onClose: PropTypes.func,
+  refHeader: PropTypes.shape({ current: PropTypes.instanceOf(Element) }).isRequired,
+  refSidePanelContent: PropTypes.shape({ current: PropTypes.instanceOf(Element) }).isRequired,
```

Comment:
`refContent`? In general, including `SidePanel` in this component's names seems redundant, and shorter names will be easier to read.

Reply:
mmm actually is the ref for the SidePanel inside of the Dialog.js component. Makes sense to specify the name.